### DISCUSSION TITLE
Is Notes connecting to external servers?

### DISCUSSION THREAD

**Discussion by luc0x61** [nuttyartist/notes#530] [General]
I suggested Notes be mentioned in a blog about outliners, but one of the users found **strange** behavior in it, trying to connect to an external server
[https://www.outlinersoftware.com/topics/viewt/10057/0/notes-note-taking-application](https://www.outlinersoftware.com/topics/viewt/10057/0/notes-note-taking-application)

Why is it connecting to raw.githubusercontent.com? Why there's no mention of it or how to stop that? 🤔

**Comment by guihkx** [comment]
There's nothing strange there, that's just part of the auto-update feature, which runs when you open the program.

It connects to this page specifically: 

- https://raw.githubusercontent.com/nuttyartist/notes/master/UPDATES.json

You can disable this feature in two ways:

1. Open Notes
2. Click on the 'three dots' menu
3. Click on 'Check For Updates'
4. Tick 'Don't notify me automatically every time there's a new update'

And the other way would be compiling Notes from source and specifying the [`-DUPDATE_CHECKER=OFF` option](https://github.com/nuttyartist/notes/blob/master/docs/build_options.md), which will completely disable any update checking features.

**Reply by nuttyartist** [reply]
👍
